src/components/Offers/New/form-components/Policies/PrivacyPolicyContent.js

```diff
@@ -0,0 +1,418 @@
+/* eslint-disable max-len */
+import { makeStyles, Typography } from "@material-ui/core";
+import React from "react";
+
+const useStyles = makeStyles({
+    bold: {
+        fontWeight: 500,
+    },
+    gray: {
+        color: "rgba(0, 0, 0, 0.54)",
+    },
+});
```

Comment:
This code is repeated here and on the terms and conditions page. Create a new file, `PoliciesStyle`, and move the code there.